### ISSUE TITLE
lasso package needs relative pathing

### ISSUE THREAD

**Issue by yomed** [marko-js/marko-migrate#1]
Previously you could set the lasso package like this:

```
<lasso-page name="page" package-path="browser.json"/>
```

With Lasso v2, the package is resovled using `require.resolve()` which will fail without a relative or absolute path specified here.

```
<lasso-page name="page" package-path="browser.json"/> // Error: Cannot find module 'browser.json'
<lasso-page name="page" package-path="./browser.json"/> // works
```

**Comment by patrick-steele-idem** [comment]
Thanks for reporting the problem. Should be fixed now. Thanks again.

**Comment by patrick-steele-idem** [comment]
New version of migration tool published: `marko-migrate@1.0.9`
